### PR TITLE
AI eyes with relay speech effects the AI with voice of god if the ai eye has relay speech

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

AI eyes (and other camera eyes) no longer runtime when VOG is used near them, breaking VOG

Instead, if it is a normal camera eye, nothing happens.

If it is a ai camera eye that has the "hear from it's camera" function, it instead will be affected as well by the VOG! Fun for the whole family!

## Why It's Good For The Game

Runtimes bad.
Funny niche interactions good!

## Testing

Used vog, no runtime, ai unaffected.
Used VOG with relay speach, AI affected.

## Changelog
:cl:
tweak: AI eyes with relay speech effects the AI with voice of god if the ai eye has relay speech
/:cl:

### CODE REVIEW

**Comment by 1080pCat** [comment]
this title makes me feel like im having a stroke

File: code/modules/surgery/organs/vocal_cords.dm [hunk 1]
```diff
@@ -166,7 +166,12 @@ GLOBAL_DATUM_INIT(multispin_words, /regex, regex("like a record baby"))
 				var/mob/living/carbon/human/H = L
 				if(H.check_ear_prot() >= HEARING_PROTECTION_TOTAL)
 					continue
-			listeners += L
+			if(istype(L, /mob/camera/aiEye))
+				var/mob/camera/aiEye/AEAIO = L
```
Comment (Contrabang) [inline code/modules/surgery/organs/vocal_cords.dm:170]: this is funny, but also confusing. Can you change this to something like `ai_eye`

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]
once Contra's reviews are addressed I'll approve

File: code/modules/surgery/organs/vocal_cords.dm (hunk 1 above)
Comment (Qwertytoforty) [reply code/modules/surgery/organs/vocal_cords.dm:170]: will do osrry forgot

**Review by Sirryan2002 — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**